openest/generate/base.py

```diff
@@ -11,7 +11,7 @@ def format(self, lang):
 
     def apply(self, region):
         def generate(region, year, temps, **kw):
-            yield (year, self.value)
+            yield year, self.value
```

Comment:
Why these? py3 handles this properly. If we do this, then we would want to do the same with tuple returns and setting from those returns.

Reply:
If I follow your concern... this should still yield a tuple, even without the `()`. We're just removing redundancy, I think.